server/queues/processors/NotificationsProcessor.ts

```diff
@@ -182,3 +225,85 @@ export default class NotificationsProcessor extends BaseProcessor {
     }
   }
 }
+
+type Notifiable = {
```

Comment:
Lots of duplicate data in this type, could just be `user`, `document` and `event`?